### DISCUSSION TITLE
iOS 16 beta users !

### DISCUSSION THREAD

**Discussion by UntrustedTech** [thebaselab/codeapp#524] [Show and tell]
I've been testing out **Code** app using the new iOS 16 private beta and public beta in multiple devices, compared to other code apps that I use for web development, code app has been the most stable of them all. It works amazing in standalone and also using the new updated second monitor support. I posted a image below don't mind the mess neither what I'm watching it's just stranger things. ![image](https://user-images.githubusercontent.com/82400546/180123181-c87dd5ab-efcf-4314-8220-49610538ba76.jpeg) The only problem I've had was importing files and stuff from external drives but _thebaselab_ already knows about it and also figured out a work around this issue. Anyways I give this app 10/10, the only thing I'm looking forward is to see if the dev for code app will implement the new features for iOS app developers like that new work system and stuff. I'm still testing out ways to break the app and as I said the 3rd party thing and some weird issue I get with sftp which is fine.

**Comment by hg0428** [comment]
> I've been testing out **Code** app using the new iOS 16 private beta and public beta in multiple devices, compared to other code apps that I use for web development, code app has been the most stable of them all. It works amazing in standalone and also using the new updated second monitor support. I posted a image below don't mind the mess neither what I'm watching it's just stranger things. ![image](https://user-images.githubusercontent.com/82400546/180123181-c87dd5ab-efcf-4314-8220-49610538ba76.jpeg) The only problem I've had was importing files and stuff from external drives but _thebaselab_ already knows about it and also figured out a work around this issue. Anyways I give this app 10/10, the only thing I'm looking forward is to see if the dev for code app will implement the new features for iOS app developers like that new work system and stuff. I'm still testing out ways to break the app and as I said the 3rd party thing and some weird issue I get with sftp which is fine.

My issues:
Code App has issues when resizing.
Code App does not support multiple windows.
Code App crashes very often.
